actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -416,6 +383,14 @@ def values
       to_enum(:each_value).to_a
     end
 
+    # Returns +true+ if the given key is present in the parameters.
+    def include?(key)
+      to_unsafe_h.include?(key)
```

Comment:
The reason we're not delegating is to account for potential hash keys. Let's make sure each key is converted to params as needed, and the specified key is too. You can reference the PR that implements `has_value?` for inspiration.

Reply:
Yes, on second thought I think you're right. The only way a hash comparison could happen is if the internal hash was manipulated, and we don't need to account for that.